src/diffusers/pipelines/stable_diffusion/convert_from_ckpt.py

```diff
@@ -1172,9 +1215,9 @@ def download_from_original_stable_diffusion_ckpt(
             checkpoint, original_config, checkpoint_path, image_size, upcast_attention, extract_ema
         )
 
-    num_train_timesteps = original_config.model.params.timesteps
-    beta_start = original_config.model.params.linear_start
-    beta_end = original_config.model.params.linear_end
+    num_train_timesteps = original_config.model.params.timesteps or 1000
+    beta_start = original_config.model.params.linear_start or 0.02
+    beta_end = original_config.model.params.linear_end or 0.085
```

Comment:
Where are these numbers coming from? I'd make a note for our future reference.

Reply:
Ah this is hacky for now and shouldn't be this way